### ISSUE TITLE
Add a copy buttons in the descriptor registration pages (or replace labels by non editable text input)

### ISSUE THREAD

**Issue by pythcoiner** [wizardsardine/liana#1164]
![image](https://github.com/Blockstream/Jade/assets/124568858/1dbf927c-28fb-4b05-a73d-6c743f348769)

**Comment by nondiremanuel** [comment]
It would also be nice to show them in a way the user doesn't need to scroll horizontally to see them all imho

**Comment by kloaec** [comment]
I tend to disagree, why should it have a copy button? The descriptor backup step is prior to that one.

Regarding label, not sure what you mean.
(and actually i think we should display the "Alias" next to the Key@ number.
